Package.swift

```diff
@@ -60,6 +240,12 @@ let package = Package(
             cSettings: [
                 .headerSearchPath("."),
                 .headerSearchPath("Monitors"),
+                .unsafeFlags(warningFlags),
+            ],
+            cxxSettings: [
+                .headerSearchPath("."),
+                .headerSearchPath("Monitors"),
+                .unsafeFlags(warningFlags),
```

Comment:
As far as I remember, `unsafeFlags` are not permitted in the release version of the SPM package.  We should come up with a way to limit them to KSCrash development only.

Reply:
Ugh... There's got to be some way to keep this working, because this actually uncovered a few bugs.

Reply:
I think you might be able to use a few `.when(configuration: .debug)` to get this only into debug mode. Not sure if that applies to this particular part of the SPM package code, but it might.